build.sh

```diff
@@ -13,17 +15,14 @@ else
 fi
 branch=$branch","$cid
 
-output=./bin/
```

Comment:
when I'm not  a root user, a compile error caused by deleting the `output` directory 
```
/usr/bin/ld: cannot open output file /hypervisor: Permission denied
collect2: error: ld returned 1 exit status
```
because the build line:
```bash
gcc -Wall -O3 scripts/hypervisor.c -o ${output}/hypervisor -lpthread
```